cli/tests/pcluster/config/test_validators.py

```diff
@@ -679,6 +679,76 @@ def _kms_key_stubber(mocker, boto3_stubber, kms_key_id, expected_message, num_ca
             "'per_unit_storage_throughput' can only be used when 'deployment_type = PERSISTENT_1'",
             0,
         ),
+        (
+            {"storage_capacity": 1200, "deployment_type": "PERSISTENT_1", "automatic_backup_retention_days": 2},
+            None,
+            None,
+            1,
```

Comment:
My understanding is that the `num_calls` value should only be nonzero if either the `export_path` or `import_path` parameter is specified, which isn't the case for the majority of these test cases. Can this be 0?